### PR TITLE
11318

### PR DESCRIPTION
**Pull request by DialloOracle**
# 11318

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x] All filenames are lower case (including folders, images, files, etc.)
- [x] Filenames are descriptive
- [x] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [ ] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [x] Image references in markdown contain an alternate text

### CODE REVIEW

**Review by anacoman11 — APPROVED** [review]

**Review by anacoman11 — APPROVED** [review]

**Review by dannymgithub — APPROVED** [review]

**Review by dannymgithub — DISMISSED** [review]

**Comment by kamryn-v** [comment]
Hi @DialloOracle , please resolve the conflicts

File: database/protect-sensitive-data-in-rest-get-calls-using-oracle-data-redaction/use-redaction-anonymize-all-restcalls-and-queries/use-redaction-anonymize-all-restcalls-and-queries.md [hunk 1]
```diff
@@ -179,31 +158,17 @@ This lab assumes you have:
...
+    ![Redacted REST](images/redacted-qry.png)
 
 Congratulations, You have successfully redacted REST calls using ORDS!
 
 ## Acknowledgements
 
 - **Authors** - Alpha Diallo & Ethan Shmargad, North America Specialists Hub
 - **Creator** - Pedro Lopes, Database Security Product Manager
-- **Last Updated By/Date** - Alpha Diallo & Ethan Shmargad, January 2023
+- **Last Updated By/Date** - Alpha Diallo & Ethan Shmargad, February 2023
```
Comment (dannymgithub) [inline database/protect-sensitive-data-in-rest-get-calls-using-oracle-data-redaction/use-redaction-anonymize-all-restcalls-and-queries/use-redaction-anonymize-all-restcalls-and-queries.md:173]: remo

Comment (dannymgithub) [inline database/protect-sensitive-data-in-rest-get-calls-using-oracle-data-redaction/use-redaction-anonymize-all-restcalls-and-queries/use-redaction-anonymize-all-restcalls-and-queries.md:173]: please removed duplicated **Last Updated By line

**Comment by DialloOracle** [comment]
Hi Camryn,

I resolved the point brought up by the comment. Are there any other steps to complete? Also when the pull request is approved when will I see the change reflected on the livelabs site?

Thanks,
Alpha

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Kamryn Vinson ***@***.***>
Sent: Wednesday, February 14, 2024 8:17 AM
To: oracle-livelabs/security ***@***.***>
Cc: Alpha Diallo ***@***.***>; Mention ***@***.***>
Subject: [External] : Re: [oracle-livelabs/security] 11318 (PR #166)


Hi @DialloOracle<https://urldefense.com/v3/__https://github.com/DialloOracle__;!!ACWV5N9M2RV99hQ!KanIzCMVf95CkFtBOGGMGR-EIg-_xEh2Ij47KPp_M_wqHuTm1NVBRb6uCcixp2Mp8b3X-epF2Sa28PIpbJ9_3uhYfA4$> , please resolve the conflicts

**Review by kamryn-v — DISMISSED** [review]

**Review by dannymgithub — DISMISSED** [review]

**Comment by ramumg** [comment]
Hi @DialloOracle, it looks like your brancch still has conflicts that need to be resolved. Please take care of them so we can go ahead and merge the changes.

**Comment by DialloOracle** [comment]
Hello, I have a bit of time now so I'll work on it.
________________________________
From: ramumg ***@***.***>
Sent: Tuesday, February 20, 2024 11:58 AM
To: oracle-livelabs/security ***@***.***>
Cc: Alpha Diallo ***@***.***>; Mention ***@***.***>
Subject: [External] : Re: [oracle-livelabs/security] 11318 (PR #166)


Hi @DialloOracle<https://urldefense.com/v3/__https://github.com/DialloOracle__;!!ACWV5N9M2RV99hQ!NnsDt13g0TLErWof7uPm4PEy6pMdohgpvAdT-DEcRmoWttsHy3VShH47Cz2z8yJ8Pswfrbx47z1yFPw8oNyzcuc_82o$>, it looks like your brancch still has conflicts that need to be resolved. Please take care of them so we can go ahead and merge the changes.

**Comment by DialloOracle** [comment]
> Hi @DialloOracle, it looks like your brancch still has conflicts that need to be resolved. Please take care of them so we can go ahead and merge the changes.

Hi Ramu, 

When I follow the rebase steps described here([https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line)) in my security repo I don't get any details on any merge conflict. When I run git status this is what I see:

aldiallo@MacBook-Pro-3 security % git status
On branch main
Your branch is up to date with 'origin/main'.

nothing to commit, working tree clean

I see the conflicts listed in the pull request thread in Github, so I assumed that they would be here listed after the status command. Any idea on how I can resolve this and complete this process? Also, I want to mention that in that conflict list in github the key vault workshop in my environment is one that I don't want to be included in the pull request, everything else in the list are files included in the workshop we want to make changes to.

Thanks,

Alpha

**Review by ramumg — APPROVED** [review]

**Pull request merged**